### PR TITLE
Update brownbag form

### LINKED ISSUE #214: Update Brownbag Idea Form
Ref: Discussions with AV Team

## Proposed solution

Add more questions to the form to collect necessary data at time of talk proposal

## Collaborators 

@danieltott 
@BekahHW

### PR DESCRIPTION
**Pull request by meg-gutshall**
## Linked Issue

Fixes #214 

## Description

This is a draft that adds the fields we'll need to cover for most of the brownbag talks. I don't know Nunjucks and I don't have access to Netlify's end, but it's a start.

There should be conditional statements that reveal different parts of the form depending upon the user's answer, I just didn't get a chance to code that out. These breakpoints are referenced in [my most recent PR in VC-Events](https://github.com/Virtual-Coffee/VC-Events/pull/125).

I ran this in preview and all looked fine!

## Methodology

The addition of these fields is to increase automation in the information gathering process (and hopefully later on, the complete workflow) of the brownbag events.

### CODE REVIEW

**Review by saramccombs — COMMENTED** [review]
I really think these changes make the brownbag form more actionable and streamlined. This lets us get more information up front and could minimize any back-and-forth conversations between the brownbag team and the presenter that could delay getting this event scheduled.

File: src/forms/brownbag-submissions.njk [hunk 1]
```diff
@@ -56,42 +56,300 @@ permalink: brownbag-idea/
...
             class="form-control"
             id="topicTitle"
             name="topicTitle"
-            required
           />
         </div>
         <div class="form-group">
-          <label for="topicIdea">Description we can share on the event page.</label>
-          <textarea class="form-control" required id="topicIdea" name="topicIdea" rows="3"></textarea>
+          <label for="topicDescription">What is the description of your talk?</label>
```
Comment (saramccombs) [inline src/forms/brownbag-submissions.njk:73]: I see the name/id changed in this commit, will this be a blocker @danieltott? Also, looks like the 'required' was dropped. Are we not requiring a description? I think something should be required for the description.. but looking at other form fields it doesn't look like we are requiring anything. Should we?

File: src/forms/brownbag-submissions.njk [hunk 2]
```diff
@@ -56,42 +56,300 @@ permalink: brownbag-idea/
...
+          <textarea
+            class="form-control"
+            id="topicDescription"
+            name="topicDescription"
+            rows="3">
+          </textarea>
+        </div>
+        <p>Please give three dates and times you're available to present in order of preference. Be sure to include timezone!</p>
```
Comment (saramccombs) [inline src/forms/brownbag-submissions.njk:81]: Looking at @BekahHW's comments, can we maybe change this to have these three options with a disclaimer that we will not do brownbags on Tuesday and Thursdays due to those being our coffee days:
1. Friday at 12:00 PM ET (the de-facto brownbag time)
2. Monday at 9:00 AM ET (the non-Noon set brownbag time)
3. Other

File: src/forms/brownbag-submissions.njk (hunk 1 above)
Comment (BekahHW) [reply src/forms/brownbag-submissions.njk:73]: My thought on this is that many people who are submitting have ideas about what they'll talk about but haven't finalized them. So requiring _some_ fields would definitely be a preventative measure. For example, requiring a bio or headshot at this stage does cut back on some of the data gathering, but for members who don't have those things available, that means they don't turn in the form. Even asking the format could be premature here IMO.

For the podcast, I send out the email and calendar invite at the same time. In the email, I let them know what they need, and what I'll need by that time. If I don't have that three days before the event, I slack them. 

So my suggestion here would be to add language to say, before the brownbag, please send us x,y, and z. Or to tack on to that, _or_ if you have them available now, here are the fields.

**Comment by danieltott** [comment]
Thanks again @meg-gutshall for all of your work here. I was able to use these ideas to create the new speaker bios feature and to inform the creation of the Events project view. Even though we're not using this PR as-is, the work you put in to this was a huge help 🙌

**Pull request closed without merging**